css/main.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+* {
```

Comment:
Is this your reset CSS file ? If yes ? 

Are you not using your CL to get this from there using your own CDN?

Reply:
I have not included reset css in my cl. Didn't knew about it. Will make the changes.